src/tweenable.rs

```diff
@@ -119,6 +121,13 @@ pub trait Tweenable<T>: Send + Sync {
     /// reach back the same state in this case is the double of the returned value.
     fn duration(&self) -> Duration;
 
+    /// Set the animation speed. Defaults to 1.
+    ///
+    /// Speeds greater than 1 slow down time. That is, a speed of 10 means the animation will
+    /// take 10 times longer to complete whereas a speed of 0.5 means the animation will complete
+    /// twice as fast.
```

Comment:
This is really counter-intuitive. Higher speeds should play the animation faster, not slower.